### PR TITLE
feat: added multisig pallet

### PR DESCRIPTION
**Pull request by lumir-mrkva**

### CODE REVIEW

**Comment by jak-pan** [comment]
LGTM if tested.

File: runtime/common/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "common-runtime"
-version = "100.0.3"
+version = "106.0.0"
```
Comment (dmoka) [inline runtime/common/Cargo.toml:3]: what is the reason for the 6 major version bumb? Keeping it in sync with hydradx-runtime and testing-hydradx-runtime?

**Review by dmoka — CHANGES_REQUESTED** [review]
Left only one comment about the version change of common-runtime. If that is intentional, then I am fine the PR. ;)

I submitted my review as "Request changes" for safety reasons, because I am not sure that if I  submit with "Approve" then it will automatically merge it to master, or it is merged only after the PR owner resolves the new comments. What is our policy here? Please let me know, thank you!

File: runtime/common/Cargo.toml (hunk 1 above)
Comment (enthusiastmartin) [reply runtime/common/Cargo.toml:3]: i believe it is just to match hydradx-runtime and testing-runtime versions. 

but it is necessary ? we dont do this in basilisk and common does not change everytime a runtime changes.

**Review by enthusiastmartin — COMMENTED** [review]
looks ok to me.

if you want to have this benchmarked and not use default weights , you might want to add this to list of pallets to benchmark ( only in hydradx-runtime )

**Review by dmoka — APPROVED** [review]

**Pull request merged**